SAPHana/bin/SAPHanaSR-manageAttr

```diff
@@ -670,31 +680,110 @@ chk_multi_target_requirements() {
     return $chk
 }
 
+# get resource name from cib
+get_res() {
+    local resType=$1
+    cib=/var/lib/pacemaker/cib/cib.xml
+    # get resources for given $RSID
+    resS="$(grep '<nvpair name="SID"' $cib | sed -n "/value=\"${RSID}\"/s/.*id=\"\(.*\)-instance_attributes-.*>/\1/gp")"
+    if [ -n "$INO" ]; then
+        # get resources for given $INO
+        resI="$(grep '<nvpair name="InstanceNumber"' $cib | sed -n "/value=\"${INO}\"/s/.*id=\"\(.*\)-instance_attributes-.*>/\1/gp")"
+    fi
+    # get cln or msl resource name for given $RSID
+    # if possible do not use 'crm configure show', even if it's nicer to parse
+    # it's much slower than direkt reading cib.xml
```

Comment:
But we also much more depend on (small) syntax changes in the cib as we have had in the past.